### PR TITLE
[ci] Added GitHub workflow for CI

### LINKED ISSUE #116: [ci] Move CI to Github actions
Move CI to Github actions as we did with other modules.

### PR DESCRIPTION
**Pull request by devkapilbansal**
Closes #116

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          libsqlite3-mod-spatialite
+
+      - name: Install python dependencies
+        run: |
+          pip install -U pip wheel setuptools
+          pip install ${{ matrix.django-version }}
+          pip install -U -r requirements-test.txt
+
+      - name: Install npm dependencies
+        run: sudo npm install -g jslint
```
Comment (Purhan) [inline .github/workflows/ci.yml:58]: Are we actually running jslint? I might be a little out of the loop :smile:

Comment (nemesisdesign) [reply .github/workflows/ci.yml:58]: thanks @Purhan, no we're not: https://github.com/openwisp/openwisp-firmware-upgrader/blob/master/run-qa-checks
there's no JS in this repo at the moment, only some CSS (would be good to run the css linter)

Comment (devkapilbansal) [reply .github/workflows/ci.yml:58]: Then it is safe to remove `.jslintrc` file too.

**Review by nemesisdesign — APPROVED** [review]
Thank you very much @devkapilbansal :+1: :+1:

**Pull request merged**